src/main/java/org/fogbowcloud/app/core/IguassuController.java

```diff
@@ -255,8 +254,8 @@ private void initMonitors() {
     }
 
     private void initJobStateMonitor() {
-        final long JOB_MONITOR_EXECUTION_PERIOD = Long.getLong(
-            this.properties.getProperty(ConfProperties.JOB_STATE_MONITOR_PERIOD));
+        final long JOB_MONITOR_EXECUTION_PERIOD = Objects.requireNonNull(Long.valueOf(
```

Comment:
Is there already a check for null previously?

Reply:
Solved in 237525f99fb1ea9d1c59a80fda501accf8dd5a98.